convert-hf-to-gguf.py

```diff
@@ -2829,19 +2975,22 @@ def main() -> None:
         "auto": gguf.LlamaFileType.GUESSED,
     }
 
-    if args.outfile is not None:
-        fname_out = args.outfile
-    else:
-        # output in the same directory as the model by default
-        fname_out = dir_model / 'ggml-model-{ftype}.gguf'
-
     logger.info(f"Loading model: {dir_model.name}")
 
     hparams = Model.load_hparams(dir_model)
 
     with torch.inference_mode():
-        model_class = Model.from_model_architecture(hparams["architectures"][0])
-        model_instance = model_class(dir_model, ftype_map[args.outtype], fname_out, args.bigendian, args.use_temp_file, args.no_lazy)
+        encodingScheme = ftype_map[args.outtype]
```

Comment:
Maybe change this to `output_type` or something like that? `encodingScheme` feels weird (also camel case used instead of snake case feels out of place). Goes for every `encodingScheme`

Reply:
@Galunid I agree with you on the code style. Better to keep styles consistent and follow typical language guidelines. Python style guide actually encourages snake case; C and C++ are similar unless the dev prefers something else.

The variable name won't matter in the future because it ideally won't exist anymore. For the interim, a better name might be `model_precision`, but `ftype` or `file_type` are fine as well. It never bothered me much.